cilium-health/launch/endpoint.go

```diff
@@ -338,15 +341,14 @@ func LaunchAsEndpoint(baseCtx context.Context,
 		}
 	}
 
-	// Set up the endpoint routes
+	// Set up the endpoint routes.
 	if err = configureHealthRouting(info.ContainerName, epIfaceName, node.GetNodeAddressing(), mtuConfig); err != nil {
 		return nil, fmt.Errorf("Error while configuring routes: %s", err)
 	}
 
 	if option.Config.IPAM == ipamOption.IPAMENI {
 		if err := routingConfig.Configure(healthIP,
-			mtuConfig.GetDeviceMTU(),
-			option.Config.Masquerade); err != nil {
+			mtuConfig.GetDeviceMTU(), masq); err != nil {
```

Comment:
I don't see any IPv6 support in `routingConfig.Configure(...)` so maybe not worth making any changes here for now?

Reply:
Correct, ENI mode does not support IPv6.